.github/workflows/ci.yml

```diff
@@ -232,12 +232,30 @@ jobs:
         run: invoke public-api
       - name: link_checker
         run: python docs/checks/docs_link_checker.py -p docs/docusaurus/docs -r docs/docusaurus/docs -s docs --skip-external
-      - name: docs_build
+
+  docs-build:
+    needs: [doc-checks, docs-changes]
+    # run on non-draft PRs with docs changes
+    if: |
+      (github.event.pull_request.draft == false && needs.docs-changes.outputs.docs == 'true') ||
+      (github.event_name == 'push' && contains(github.ref, 'refs/tags/'))
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: "3.8"
+          cache: "pip"
+          cache-dependency-path: reqs/requirements-dev-test.txt
+      - name: Install dependencies
+        run: pip install -r reqs/requirements-dev-test.txt
+      - name: Build docs
```

Comment:
@anthonyburdi This step is currently failing on this PR.
Is there something I need to add to this step or something else going on?

Reply:
Thanks, that seems to have been it.